wildfly/extensions/src/main/java/org/keycloak/provider/wildfly/JsonConfigProviderFactory.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Remove this. We haven't supported keycloak-server.json in the WF distribution for a very long time.